guides/development-workflow.md

```diff
@@ -474,3 +475,19 @@ A new registration form and fix a bug with contacts
 - Passwords are encrypted using BCrypt, so I've taken this opportunity to
   increase the default rounds from 8 to 12
 ```
+
+## Versioning releases
+
+We don't use explicit versioning for frontend projects; rather we operate a continuous deployment approach as described in the [deploying](#deploying) section.
```

Comment:
Do we have a style guide for frontend vs front-end vs front end? (Should we?)

Reply:
I'm not totally happy with "frontend projects" here anyway. I mean "code that isn't reused by other code", but I couldn't think how to put it...

Reply:
I'm not super-fussed tbh, but others might have an opinion 😆

Reply:
"apps", "services", "non-libraries"? I would read "frontend" are referring specifically to JS, CSS, HTML if I was reading this without full dxw context.

Reply:
I agree @erbridge, that's what I don't like about it. What about "client projects"?

Reply:
What about "consumer facing apps"? I'm trying to think of a way to express that we're talking about stuff that is on the web and people use.

Reply:
The question is "Does anything need to depend on this thing? If not, then we don't version it. Maybe it could be rephrased to just make that distinction?

Reply:
The consenses of my question on twitter was that "application code" is probably the right thing, so I've changed it to that.